src/foam/lib/json/Outputter.java

```diff
@@ -455,4 +493,12 @@ public void flush() throws IOException {
     if ( stringWriter_ != null ) stringWriter_.flush();
     if ( writer_ != null ) writer_.flush();
   }
+
+  public void makeMultiLine() {
```

Comment:
Could make this follow the standard get/set pattern that most properties follow.